iceoryx_posh/test/moduletests/test_roudi_memory_provider.cpp

```diff
@@ -64,14 +73,49 @@ class MemoryProvider_Test : public Test
         sut.addMemoryBlock(&memoryBlock1);
         EXPECT_CALL(memoryBlock1, sizeMock()).WillRepeatedly(Return(COMMON_SETUP_MEMORY_SIZE));
         EXPECT_CALL(memoryBlock1, alignmentMock()).WillRepeatedly(Return(COMMON_SETUP_MEMORY_ALIGNMENT));
-        EXPECT_CALL(sut, createMemoryMock(COMMON_SETUP_MEMORY_SIZE, COMMON_SETUP_MEMORY_ALIGNMENT)).Times(1);
+    //    EXPECT_CALL(sut, createMemoryMock(COMMON_SETUP_MEMORY_SIZE, COMMON_SETUP_MEMORY_ALIGNMENT)).Times(1);
 
-        EXPECT_CALL(sut, destroyMemoryMock());
+    //    EXPECT_CALL(sut, destroyMemoryMock());
         EXPECT_CALL(memoryBlock1, destroyMock());
 
         return sut.create();
     }
 
+    static const int16_t nTestCase = 13;
```

Comment:
``int16_t`` is ok, but it really buys us nothing here, could be ``int32_t`` or ``uint32_t`` as well. Note that smaller types can even be detrimental wrt. execution time when the native machine word length is larger (due to the way the hardware has to access those bytes).